README.md

```diff
@@ -58,7 +58,9 @@ To deploy this service into a new account:
   1. Prepare your account. The following SSM parameters need to exist:
      - `/account/services/artifact.bucket`
      - `/account/services/logging.stream`
-  2. Raise a PR, adding your stack to [`packages/cdk/bin/cdk.ts`](packages/cdk/bin/cdk.ts)
+  2. Raise a PR, adding your [deployment-resource-stack](https://riffraff.gutools.co.uk/deployinfo/data?key=credentials%3Aaws-cfn-role) to [`packages/cdk/bin/cdk.ts`](packages/cdk/bin/cdk.ts):
```

Comment:
Somehow, this URL works for me when pasting it directly in the address bar, but I get a 403 forbidden if I [try to follow the link](https://github.com/guardian/cloudwatch-logs-management/blob/c38db7c372ba8a3720b6008d276e32681351ec08/README.md#deployment) from github. Does it work for you? I'm wondering if it's something related to the referral

<img width="775" alt="Screenshot 2024-08-07 at 16 15 23" src="https://github.com/user-attachments/assets/bb2b69a9-bbea-4260-a34b-3fa7991878c5">

Reply:
Hmmm - it works for me, but it might be that I am logged into riffraff and thats why? Thats going to be confusing. I'll get rid of the link and add something like "(See the deployment-resources tab in riffraff for examples)" instead

Reply:
Added a new commit with a link that goes to riffraff home page. Hopefully they will figure it out by the name of the tab there.